oeps/processes/oep-0056-architectural-advisory-process.rst

```diff
@@ -0,0 +1,197 @@
+======================================
+OEP-56: Architectural Advisory Process
+======================================
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-56 <oep-0056-architectural-advisory-process>`
+   * - Title
+     - Architectural Advisory Process
+   * - Last Modified
+     - 2022-02-25
+   * - Authors
+     - David Joy <djoy@edx.org>
+   * - Arbiter
+     - Feanil Patel <feanil@tcril.org>
+   * - Status
+     - Accepted
```

Comment:
The [wording for Provisional status](https://github.com/openedx/open-edx-proposals/blob/802c833a0c5369c8a5995e2d951dfb7707a1795d/oeps/processes/oep-0001.rst#provisional) isn't exactly about process OEPs, but it could be considered here. It doesn't wildly change anything, except to let people new to the platform know that this process is new to all of us.

Reply:
A good point.  What do you think @feanil?

Reply:
As I think this through, I like the idea of starting with Provisional even more.  I'm somewhat uncomfortable with the idea of merging this PR and then just saying "Go follow the process!" - we need to invest in issue/PR templates, the checklist, instructions, and evangelization it a bit before we can expect it to be useful.

Reply:
Agree, let's start with Provisional for now and we can workout a schedule for when to re-visit moving from provisional to accepted.